scvi/distributions/_negative_binomial.py

```diff
@@ -236,6 +238,25 @@ def _gamma(theta, mu):
     return gamma_d
 
 
+class Poisson(PoissonTorch):
+    """
+    Poisson distribution.
+
+    Parameters
+    ----------
+    rate
+        rate of the Poisson distribution.
+    validate_args
+        whether to validate input.
+    scale
+        Normalized mean expression of the distribution.
```

Comment:
it should be noted that this is not used in any computations, same in the other distributions

Reply:
like it's only used for caching the value basically for later

Reply:
Done!